src/styles/style.css

```diff
@@ -0,0 +1,238 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+}
+
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #eee;
+  height: 100px;
+  width: 100px;
+  border: 1px solid black;
+  animation: translateY(-20px);
```

Comment:
The `animation` property seems to be incorrectly used here. It's missing the `@keyframes` rule that defines the animation. You should define a `@keyframes` rule with the name `translateY` and specify the stages of the animation. For example:

@keyframes translateY {
  from {
    transform: translateY(0);
  }
  to {
    transform: translateY(-20px);
  }
}

Then you can use it in your `.calendar__day` class like this:

.calendar__day {
  ...
  animation: translateY 0.5s ease-in-out;
}